main.cpp

```diff
@@ -38,18 +41,53 @@ int main(int argc, char *const argv[]) {
     }
 
     const IR::P4Program *program = nullptr;
-
+    // std::cout << "\nChecking " << options.file;
+    cstring command = "python3 /home/roborobo/projects/gauntlet_paper/gauntlet/"
```

Comment:
Instead of a hardcoded path I would recommend making the file a command line argument that can be parsed. Something like --validator-binary